test/SourceKit/NameTranslation/basic.swift

```diff
@@ -66,6 +67,7 @@ class MyDerived: FooClassDerived {
 // CHECK9: fooProperty11
 // CHECK10: init(nstanceFunc21:)
 // CHECK11: init(lassDerived2:)
+// CHECK11-1: FooClassDerived2
```

Comment:
This should only happen on Objc->Swift, right? where cursor is selected by indexer at a random place. For the other direction, we actually have the cursor user points to,
so we don’t need this modification.